plotly/tests/test_core/test_tools/test_TraceFactory.py

```diff
@@ -0,0 +1,89 @@
+from unittest import TestCase
```

Comment:
:cow2: super minor, but for consistency (it's also the consistency cow) none of the test names have TileCase could we `test_TraceFactory.py` --> `test_trace_factory.py` :smiley_cat: